### ISSUE TITLE
Conflict with the background-geolocation plugin

### ISSUE THREAD

**Issue by AmineCabbers** [dpa99c/phonegap-launch-navigator#22]
In my application I need two plugins (background-geolocation and launch-navigator), which have a dependency with the geolocation plugin.
I use cordova 4.3.1, suddenly, the backgroud-geolocation plugin will install the dependency plugin org.apache.cordova.geolocation but the launch-navigator plugin will install the plugin cordova-plugin-geolocation

Following this conflict, I have build problems with the following error:

*ERROR building one of the platforms: Error: / Tests / hello / platforms / ios / cordova / build: Command failed with exit code 2
You May not-have the required environment or OS to build this project
Error: / Tests / hello / platforms / ios / cordova / build: Command failed with exit code 2
    Has ChildProcess.whenDone (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/cordova/superspawn.js:131:23)
    Has ChildProcess.emit (events.js: 110: 17)
    Has maybeClose (child_process.js 1008: 16)
    Has Process.ChildProcess._handle.onexit (child_process.js: 1080: 5)*

Could you help me please?

**Comment by dpa99c** [comment]
The reason I added the dependency is that iOS and Windows need the geolocation plugin to reliably retrieve the start location if it is not provided to the plugin. I hadn't considered that other plugins would have a dependency on a different geolocation plugin. So I've removed the dependency on the geolocation plugin and made installing it a recommendation in the documentation

So if you remove the plugin from your project and then install the latest version (2.8.0), this should resolve your issue.

Let me know if that doesn't fix your problem.

**Comment by AmineCabbers** [comment]
Thank you for your quick response.
I was able to fix the problem on my side too.
Here's what I did:
I add the two plugins.
I change the file hello/plugins/launch-navigator/plugins.xml this way:
I replace **<dependencyid = "cordova-plugin-geolocation"** to **<dependency id = "org.apache.cordova.geolocation"**  for cordova 4.3.1

If I may advise you another solution and do the same thing done for the background-geolocation plugin:
For those using cordova 5.0.0 they will launch the command:
*cordova plugin add https://github.com/christocracy/cordova-plugin-background-geolocation.git*
This will contain dependency with cordova-geolocation-plugin plugin's

And those using 4.x, they will launch the command:
cordova plugin add https://github.com/christocracy/cordova-plugin-background-geolocation.git#cordova-4.x
This will contain dependency plugin org.apache.cordova.geolocation

I hope I have helped you and thank you for your help;)
